cli/lib/util.js

```diff
@@ -77,6 +78,14 @@ const util = {
     }
   },
 
+  getNode11WindowsFix () {
+    if (process.version>'v11' && process.platform==='win32') {
```

Comment:
https://nodejs.org/api/process.html#process_process_version
Process.version string, and it matches with any version from 'v11'

Reply:
The problem is that in Javascript, the string 'v8' is greater than 'v11', so this would wrongly be `true` in Node 8 (`'v8' > 'v11' === true`). It's okay though, I'll fix this and get this merged.